test/e2e/info_test.go

```diff
@@ -291,4 +291,52 @@ var _ = Describe("Podman Info", func() {
 		Expect(info).ToNot(ExitCleanly())
 		podmanTest.StartRemoteService() // Start service again so teardown runs clean
 	})
+
+	It("podman startup: ensure database checks for graph driver accommodate empty string", func() {
```

Comment:
Actually this will never work because all the e2e test pass all option via cli such as --storage-driver so it not matter what we set in the config.

I think this might have to be in test/system and you pass your own --root/-runroot/--tmpdir and just leave the driver to the storage .conf